packages/eslint-plugin/src/utils/configHelpers.js

```diff
@@ -119,4 +119,17 @@ module.exports = {
       },
     ];
   },
+
+  findGitRoot: () => {
+    let cwd = process.cwd();
+    const root = path.parse(cwd).root;
+    while (cwd !== root) {
+      // .git is usually a folder, but it's a file in worktrees
+      if (fs.existsSync(path.join(cwd, '.git'))) {
+        return cwd;
```

Comment:
**suggestion:** I'd just terminate the loop with `return` , returning a value for no purpose is rather confusing

Reply:
Not sure what you mean by returning a value with no purpose? Or did you mean to say terminate with `break`?

Reply:
```diff
if (fs.existsSync(path.join(cwd, '.git'))) {
-        return cwd;
+        return;
```

Reply:
That won't work, the function needs to return a string.